<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1593</width>
    <height>917</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonAddProject">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>211</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить проект</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonOverdue">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>800</y>
      <width>351</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Узнать кто не сдал в срок задание</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listProjects">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>1071</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameExecutor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО Исполнителя:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameProject">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Название проекта:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTermTask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Срок выполнения:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDateIssuingTask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата выдачи задания:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameSupervisor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО Руководителя:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTaskProject">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Задание в рамках проекта:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNameProject">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTaskProject">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNameExecutor">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNameSupervisor">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>251</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDateIssuingTask">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>241</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTermTask">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDateHandingTask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата сдачи:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDateHandingTask">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>341</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonTaskExecutor">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>750</y>
      <width>351</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Задaчи сотрудника</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonExecutorsProject">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>750</y>
      <width>351</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Все сотрудники проекта</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonListSupervisor">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>750</y>
      <width>351</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Список проектов руководителя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineListSupervisor">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>710</y>
      <width>351</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInputNameSupervisor">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>680</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите ФИО Руководителя:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineListTaskExecutor">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>710</y>
      <width>351</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineListExecutorProject">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>710</y>
      <width>351</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelInputNameExecutor">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>680</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите ФИО Сотрудника:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInputNameTask">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>680</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите название проекта:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonOpenFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>510</y>
      <width>211</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSaveFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>550</y>
      <width>211</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите данные о проекте на английском</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1593</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
